android-auto-app/build.gradle

```
@@ -75,6 +75,16 @@ dependencies {
     implementation("com.mapbox.navigation:ui-dropin:2.10.0-rc.1")
     implementation("com.mapbox.search:mapbox-search-android:1.0.0-beta.42")
 
+    // Support libraries
```

```suggestion
    // Libraries used for showcasing android auto examples with a mobile app
```

The example here is also trying to demonstrate a minimal set of dependencies. So I think we should have a comment that speaks a little higher level. "Support libraries" is not clear about why they are needed

---

Or you could delete this comment and move all the dependencies below this other comment

`// Dependencies needed for this example.`

---

Am I missing something? I thought that `android-auto-app` is our internal development app for the AA and that the `android-auto-app` in the **mapbox-navigation-android-examples** repo is a public example that demonstrates a minimal set of dependencies.

---

`android-auto-app` is for integrating `beta` and experimental features. Experimental features and `beta` sdks are still used by customers, so i'd like to continue to present the solution for customers.

---

Done